presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -157,6 +157,15 @@
      */
     void createTable(Session session, String catalogName, ConnectorTableMetadata tableMetadata, boolean ignoreExisting);
 
+    /**
+     * Unstable!
```

Comment:
nit: Maybe just say `This SPI is unstable and subjective to change in the future` at the end of Java doc? ditto for `ConnectorMetadata`